tests/test_init.py

```
@@ -620,14 +652,25 @@ async def test_reload_propagates_pending_startup_import_failure(
     """Fail a reload when the startup lifecycle work it awaits fails.
 
     Args:
-        hass: Home Assistant instance that hosts the integration.
+        hass (HomeAssistant): Home Assistant instance that hosts the integration.
     """
     yaml_config = {DOMAIN: {"failed_startup": {CONF_VALUE: "unused"}}}
     import_started = asyncio.Event()
     allow_failure = asyncio.Event()
 
     async def failing_async_init(*args: Any, **kwargs: Any) -> Any:
```

Ruff (0.16.1)

[warning] Unused function argument: `args`

(ARG001)